python-package/mlbox/preprocessing/reader.py

```diff
@@ -306,45 +326,59 @@ def train_test_split(self, Lpath, target_name):
 
         if (type(Lpath) != list):
 
-            raise ValueError("You must specify a list of paths to load all the data")
+            raise ValueError("You must specify a list of paths "
+                             "to load all the data")
 
         elif (self.to_path is None):
 
-            raise ValueError("You must specify a path to save your data and make sure your files are not already saved")
+            raise ValueError("You must specify a path to save your data "
+                             "and make sure your files are not already saved")
 
         else:
 
-            ######################################################
-            ################## reading the files ##################
-            ######################################################
+            ##############################################################
+            #                    Reading the files
+            ##############################################################
 
             for path in Lpath:
 
-                ### reading each file ###
-                df = self.clean(path, date_strategy="complete", drop_duplicate=False)
+                # Reading each file
+
+                df = self.clean(path,
+                                date_strategy="complete",
+                                drop_duplicate=False)
+
+                # Checking if the target exists to split into test and train
 
-                ### checking if the target exists to split into test and train
                 if (target_name in df.columns):
 
-                    df_train[path] = df[df[target_name].isnull() == False].drop(target_name, axis=1)
-                    df_test[path] = df[df[target_name].isnull() == True].drop(target_name, axis=1)
-                    y_train[path] = df[target_name][df[target_name].isnull() == False]
+                    is_null = df[target_name].isnull()
+
+                    df_train[path] = df[~is_null].drop(target_name, axis=1)
+                    df_test[path] = df[is_null].drop(target_name, axis=1)
+                    y_train[path] = df[target_name][is_null]
```

Comment:
Looks like you rather want `y_train[path] = df[target_name][~is_null]`
Otherwise, LGTM